mne/channels/layout.py

```diff
@@ -445,10 +445,10 @@ def find_layout(info, ch_type=None, exclude='bads'):
         layout_name = 'magnesWH3600'
     elif has_CTF_grad:
         layout_name = 'CTF-275'
-    elif n_kit_grads <= 157:
-        layout_name = 'KIT-157'
-    elif n_kit_grads > 157:
-        layout_name = 'KIT-AD'
+    elif n_kit_grads > 0:
+        layout_name = _find_kit_system(info, n_kit_grads)
+        if layout_name == 'KIT-UMD':
+            return _auto_layout(info)
```

Comment:
why not adding a layout file for KIT-UMD? by doing this we could share the file between software.